src/main/resources/ValidationMessages.properties

```diff
@@ -1,4 +1,4 @@
-enterLfpDetails.companyNumber.wrongLength=You must enter your full eight character company number
+enterPpsDetails.companyNumber.wrongLength=You must enter your full eight character company number
 
-enterLfpDetails.penaltyNumber.emptyField = You must enter a penalty reference
-enterLfpDetails.penaltyNumber.wrongLength = Enter your penalty reference exactly as shown on your penalty letter
\ No newline at end of file
+enterPpsDetails.penaltyNumber.emptyField = You must enter a penalty reference
```

Comment:
Since we touched these 2 lines, similar to the line above, I think the whitespace can be removed around the '='